### PR TITLE
Replace deprecated DefaultCsrfProvider with CsrfTokenManager

### PR DESCRIPTION
**Pull request by ldath**

### CODE REVIEW

File: src/setup.php [hunk 1]
```diff
@@ -22,8 +20,8 @@
 define('VENDOR_TWIG_BRIDGE_DIR', VENDOR_DIR . '/symfony/twig-bridge');
 define('VIEWS_DIR', realpath(__DIR__ . '/../views'));
 
-// Set up the CSRF provider
-$csrfProvider = new DefaultCsrfProvider(CSRF_SECRET);
+// Set up the CSRF token
+$csrfToken = new CsrfTokenManager();
```
Comment (webmozart) [inline src/setup.php]: Can you name this `$tokenManager`?

Comment (webmozart) [inline src/setup.php]: or `$csrfTokenManager`, if you prefer

**Comment by ldath** [comment]
Done

**Comment by webmozart** [comment]
Thanks! :)

**Pull request merged**